orttraining/orttraining/training_ops/cpu/tensor/slice_grad.h

```diff
@@ -21,10 +21,10 @@ class SliceGrad final : public OpKernel, public SliceBase {
   template <typename T>
   Status ComputeImpl(OpKernelContext* ctx,
                      Tensor& output_grad_tensor,
-                     const std::vector<int64_t>& output_dims,
-                     std::vector<int64_t>* flattened_output_dims,
-                     const std::vector<int64_t>& starts,
-                     const std::vector<int64_t>& steps) const;
+                     const gsl::span<const int64_t>& output_dims,
+                     TensorShapeVector* flattened_output_dims,
```

Comment:
Be careful on such changes.
The return type of gsl::span::size() is sometimes signed(our version), sometimes unsigned(the latest official version). Here You are changing an unsigned type to signed.

Reply:
`size()` is unsigned in our version. `ssize()` is signed.